openmc/model/model.py

```diff
@@ -324,6 +325,28 @@ def init_lib(self, threads=None, geometry_debug=False, restart_file=None,
         # communicator
         openmc.lib.init(args=args, intracomm=intracomm, output=output)
 
+    def sync_dagmc_universe(self):
+        """
+        Synchronize all DAGMC universes with the current geometry.
+        This method iterates over all DAGMC universes in the geometry and
+        synchronizes their cells with the current material assignments.
+
+        .. versionadded:: 0.15.1-dev
+
+        Returns:
+            None
+        """
```

Comment:
No need to include this if there's no return value.

```suggestion
        """
```